services/rfq/relayer/relapi/withdraw.go

```diff
@@ -0,0 +1,33 @@
+package relapi
+
+import (
+	"context"
+	"fmt"
+)
+
+// Withdrawer is a wrapper around the RelayerClient's Withdraw method which allows
+// a relayer to withdraw at once multiple ERC20s or the native token.
+type Withdrawer interface {
+	Withdraw(ctx context.Context, withdrawRequests WithdrawRequest) (*WithdrawResponse, error)
+}
+
+type withdrawerImpl struct {
+	client RelayerClient
+}
+
+// NewWithdrawer creates a new Withdrawer.
+func NewWithdrawer(c RelayerClient) Withdrawer {
+	return &withdrawerImpl{
+		client: c,
+	}
+}
+
+// TODO: support multiple withdraw requests in one cli command (via config?)
+// Withdraw withdraws the given amount of tokens to the given address.
+func (w *withdrawerImpl) Withdraw(ctx context.Context, withdrawRequest WithdrawRequest) (*WithdrawResponse, error) {
```

Comment:
What's the point of abstracting this out? Even if you were going to do a seperate interface still no reason for this